### PR TITLE
Add template for AKS deployment

### PR DESCRIPTION
**Pull request by vyta**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by vyta** [comment]
Added .ci_skip due to dependency on service principal

File: 101-aks/azuredeploy.parameters.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+        "resourceName": {
+			"value": "GEN-UNIQUE"
+		},
+		"location":{
```
Comment (bmoore-msft) [inline 101-aks/azuredeploy.parameters.json]: remove this and use the defaultValue

File: 101-aks/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "resourceName": {
+			"value": "GEN-UNIQUE"
+		},
+		"location":{
+			"value": "eastus"
+		},
+		"dnsPrefix": {
+			"value": "GEN-UNIQUE"
+		},
+		"agentVMSize": {
```
Comment (bmoore-msft) [inline 101-aks/azuredeploy.parameters.json]: remove this and use the defaultValue

File: 101-aks/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+			"value": "Standard_D2_v2"
+		},
+		"linuxAdminUsername": {
```
Comment (bmoore-msft) [inline 101-aks/azuredeploy.parameters.json]: GEN-UNIQUE

File: 101-aks/metadata.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (bmoore-msft) [inline 101-aks/metadata.json]: add $schema property

File: 101-aks/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Azure Container Service (AKS)
+
```
Comment (bmoore-msft) [inline 101-aks/README.md]: add deploy button

File: 101-aks/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,139 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceName": {
```
Comment (bmoore-msft) [inline 101-aks/azuredeploy.json]: add a defaultValue

**Pull request merged**